.cloudbees/testing/action.yml

```diff
@@ -54,6 +59,14 @@ runs:
       run: |
         set -uo pipefail
 
+        if echo "${{ inputs.chart-location }}" | grep -q '^oci://'; then
+          if [ "${CLOUDBEES_REGISTRIES_CONFIG}" != '' -a -s "${CLOUDBEES_REGISTRIES_CONFIG}" ]; then
```

Comment:
> if [ "${CLOUDBEES_REGISTRIES_CONFIG}" != '' -a -s "${CLOUDBEES_REGISTRIES_CONFIG}" ]; then

can be shortened to
```
if [ "$CLOUDBEES_REGISTRIES_CONFIG" ]; then
```

Reply:
Actually we are always setting the scope variable `cloudbees.registries`, see [here](https://github.com/calculi-corp/dsl-engine-cli/blob/main/internal/runtime/scopes.go#L44), and if there are no registry mirrors configured then this context var would be set but the file would not exist, which was the reason why I added the check the way i did.

Reply:
But the env var will always be set - it may be empty only when the user explicitly overwrites the `registry-configuration` input with an empty string. The condition `if [ "$CLOUDBEES_REGISTRIES_CONFIG" ]` would always evaluate to true, unless the variable is empty.
When the user provides an invalid file path, we should return a clear error instead of letting the condition evaluate to false and silently ignore the invalid file path. Doing the latter is just hiding the problem from the user, making it harder for users to spot their mistake.

Reply:
Oh and the registry configuration file always exists. the contained registry list may just be empty. Otherwise this would require an unnecessary special case in every involved component run within a workflow.

Reply:
Ok - updated the check to only verify if the env var has a value, if there are any issues with the registry config file they will be reported by the `registry-config` binary call.